services/web/server/src/async_sio.py

```diff
@@ -73,6 +77,60 @@ def connect(sid, environ):
     await SIO.emit('stopJupyter', data=result, room=sid)
 
 
+@SIO.on('presignedUrl')
+async def retrieve_url_for_file(sid, data):
+    _LOGGER.debug("client %s requests S3 url for %s", sid, data)
+    try:
+        public_url = 'play.minio.io:9000'
```

Comment:
All configuration variables should go into ``source/config.py`` and then imported here. Notice in that folder, that variables are taken from environment variable (that is how the docker passes the info)